src/test/java/integration/ShipmentControllerIT.java

```diff
@@ -305,14 +309,14 @@ public void updateShipment() throws Exception {
         then().
                 body("barcode", equalTo(shipmentDtoBeforeUpdate.getBarcode())).
                 statusCode(SC_OK);
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         // check updated data
         Shipment updatedShipment = shipmentService.getEntityByUuid(shipmentUuid, user);
         ShipmentDto shipmentDto = shipmentMapper.toDto(updatedShipment);
-        long timeModified = updatedShipment.getLastModified().getTime();
+        LocalDateTime timeModified = updatedShipment.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).